src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstiertodeliusupdate/service/TelemetryEventType.kt

```diff
@@ -2,5 +2,6 @@ package uk.gov.justice.digital.hmpps.hmppstiertodeliusupdate.service
 
 enum class TelemetryEventType(val eventName: String) {
   TIER_UPDATE_CALL_SUCCESS("TierUpdateCallSuccess"),
-  TIER_UPDATE_CALL_FAILED("TierUpdateCallFailed");
+  TIER_UPDATE_CALL_FAILED("TierUpdateCallFailed"),
+  TIER_UPDATE_CALL_FAILED_OFFENDER_NOT_FOUND("TierUpdateCallFailedOffenderNotFound");
```

Comment:
This is used in dashboard reporting at https://portal.azure.com/#@nomsdigitechoutlook.onmicrosoft.com/dashboard/arm/subscriptions/a5ddf257-3b21-4ba9-a28c-ab30f751b383/resourcegroups/dashboards/providers/microsoft.portal/dashboards/aa56a54d-b4c0-443c-a5d9-2c8bfb68f06b